package.json

```diff
@@ -12,7 +12,7 @@
   },
   "scripts": {
     "lint": "eslint karma.conf.js lib test",
-    "test": "./node_modules/karma/bin/karma start --single-run --browsers PhantomJS",
+    "test": "./node_modules/karma/bin/karma start --single-run --browsers ChromeHeadless",
```

Comment:
Only for test cases we are using chromeheadless browser in this task only. My thought is like why we can't create the separate task to update the phantomJS to ChromeHeadless. Atleast to track it we should have ticket created for it. @soumen-saniel your thoughts